src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -41,48 +41,27 @@ module("dialogs-opening_or_not", {setup: prepareUnitTestModule});
  */
 /* jshint latedef: nofunc */
 function dialogTestHelper(args) {
-    // Sinon doesn't allow wrapping of `window.open` with a spy because `open`
-    // doesn't look like a function in these browsers.
-    var sinonCantWrapWindowOpen = jQuery.browser.msie &&
-        jQuery.browser.versionNumber <= 8;
+    var wymeditor = jQuery.wymeditors(0);
```

Comment:
> it may, in the future, be extended to do some more things

And at that time, I will probably recommend splitting it into another function, or at least a wrapper, so that the name can be useful. 

> When you come and read along, is dialogTestHelper not good enough to help you understand that it helps test dialogs?

Tests what about dialogs, though. That's the question you need to understand what the test is actually doing and where the work is being abstracted. The test names don't make it clear that the keyboard shortcuts and mouse clicks are both being tested. That's OK, as long as a read-through of the test code makes it clear (since the test code is concise).

> which is likely to represent the core functionality of this helper, even if changes are made?

Making name recommendations based mostly on the future is a dicey prospect. Future us will have a lot more information and ability to make a decision on a good name, so let's trust ourselves to make that decision. Current us has a function that does a lot, but that can be described in a way that the reader doesn't need to read the implementation to understand what's happening in the test. We should use that descriptive name.

docstrings are for the why and for the details applicable to someone who wants to use a function. As a reader, it's the name of a thing that is responsible for conveying what a function does.

I think you understand where I'm coming from, so this name is your call, now.